### DISCUSSION TITLE
RLS Policy - Enable delete access for users based on their user ID

### DISCUSSION THREAD

**Discussion by kevenleone** [supabase/supabase#14044] [Questions] [answered]
Hey guys, I'm using this template "Enable delete access for users based on their user ID *"

It generates the policy:

```
CREATE POLICY "policy_name"
ON public.Folders
FOR DELETE USING (
  auth.uid() = user_id
);
```

My table doesn't have the "user_id" relationship, actually it's called "userId"

Then I updated the Policy to

```
CREATE POLICY "Enable delete for users based on user_id" ON "public"."Folders"
AS PERMISSIVE FOR DELETE
TO public
USING (auth.uid() = userId)
```

A toast error message appears

<img width="344" alt="image" src="https://user-images.githubusercontent.com/22279592/235358197-52b8b205-42b2-4379-9875-445dd8c6aa40.png">

My table called "Folders" contains the userId column related to users schema

<img width="674" alt="image" src="https://user-images.githubusercontent.com/22279592/235358238-c5f959ab-bf0a-41b5-9de1-9c347ca0b04f.png">

----

I also tried to rename it to `user_id` as the template suggests and works.

Am I doing something wrong? It looks likes a bug, but I was afraid is something I don't know/found in the documentation.

**Comment by GaryAustin1** [comment, marked as answer]
You should not use uppercase in Postgres for table or column names.  When you do you must ALWAYS put them in double quotes... you will forget.
[Why is my camelCase name not working in Postgres functions or RLS policies? ](https://github.com/orgs/supabase/discussions/12893)

**Reply by kevenleone** [reply]
Good to know

Thanks @GaryAustin1 😊
